.github/workflows/coverity.yaml

```diff
@@ -0,0 +1,48 @@
+name: Coverity
+on:
+#  schedule:
+#    # run daily 20:00 on master branch
+#    - cron: '0 20 * * *'
+  push:
+    branches:
+      - coverity_scan
+jobs:
+  coverity:
+    name: Coverity
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [12.3]
+        os: [ubuntu-18.04]
+        build_type: [Release]
```

Comment:
Should probably run it both under Release and Debug build. Release might accidentally hide subtle issues that lower optimization levels expose.

Reply:
This is static analysis so compiler flags dont matter

Reply:
Right, but doesn't the build type also set certain pre-processor flags?